[2025-03-25T19:46:32.016+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-25T19:46:32.022+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: teste_ingestao_dados_fakes.gerar_dados_fakes manual__2025-03-25T19:46:28.952456+00:00 [queued]>
[2025-03-25T19:46:32.024+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: teste_ingestao_dados_fakes.gerar_dados_fakes manual__2025-03-25T19:46:28.952456+00:00 [queued]>
[2025-03-25T19:46:32.024+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-25T19:46:32.031+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): gerar_dados_fakes> on 2025-03-25 19:46:28.952456+00:00
[2025-03-25T19:46:32.037+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=106) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-25T19:46:32.038+0000] {standard_task_runner.py:72} INFO - Started process 108 to run task
[2025-03-25T19:46:32.038+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'teste_ingestao_dados_fakes', 'gerar_dados_fakes', 'manual__2025-03-25T19:46:28.952456+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/teste_ingestao_dag.py', '--cfg-path', '/tmp/tmpk6ymrdej']
[2025-03-25T19:46:32.040+0000] {standard_task_runner.py:105} INFO - Job 59: Subtask gerar_dados_fakes
[2025-03-25T19:46:32.124+0000] {task_command.py:467} INFO - Running <TaskInstance: teste_ingestao_dados_fakes.gerar_dados_fakes manual__2025-03-25T19:46:28.952456+00:00 [running]> on host 90bd565f4613
[2025-03-25T19:46:32.174+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='teste_ingestao_dados_fakes' AIRFLOW_CTX_TASK_ID='gerar_dados_fakes' AIRFLOW_CTX_EXECUTION_DATE='2025-03-25T19:46:28.952456+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-25T19:46:28.952456+00:00'
[2025-03-25T19:46:32.176+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-25T19:46:32.179+0000] {python.py:240} INFO - Done. Returned value was: [{'observation_id': 1, 'species_guess': 'Capivara', 'observed_on': '2025-03-25', 'latitude': -23.55, 'longitude': -46.63}, {'observation_id': 2, 'species_guess': 'Tamandu√°', 'observed_on': '2025-03-24', 'latitude': -15.78, 'longitude': -47.93}]
[2025-03-25T19:46:32.188+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-25T19:46:32.188+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=teste_ingestao_dados_fakes, task_id=gerar_dados_fakes, run_id=manual__2025-03-25T19:46:28.952456+00:00, execution_date=20250325T194628, start_date=20250325T194632, end_date=20250325T194632
[2025-03-25T19:46:32.222+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-25T19:46:32.249+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-25T19:46:32.250+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
